solanches/models.py

```diff
@@ -79,3 +79,31 @@ def get_all():
     def to_dict(self):
         comercio = vars(self).copy()
         return comercio
+
+
+class Cardapio:
+
+    def __init__(self, comercio_id, produtos, destaques):
```

Comment:
É bom colocar como valor padrão listas vazias em produtos e destaques, para não serem obrigatórios
```suggestion
    def __init__(self, comercio_id, produtos=[], destaques=[]):
```

Reply:
Feito!